src/pods/test-fill-gap/components/show-results.component.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
-import { Verb } from "../test-fill-gap.vm";
+import { Verb } from '../test-fill-gap.vm';
 
 interface Props {
-	isCorrect: boolean;
-	verb: Verb;
+  isCorrect: boolean;
+  verb: Verb;
 }
 
 export const ShowResultsComponent: React.FC<Props> = props => {
```

Comment:
Is this component in use or has it been replaced? if it's not in use let's remove it from the base code